awslambdaric/bootstrap.py

```diff
@@ -335,7 +335,8 @@ def __exit__(self, exc_type, exc_value, exc_tb):
 
     def log(self, msg):
         encoded_msg = msg.encode("utf8")
-        log_msg = self.frame_type + len(encoded_msg).to_bytes(4, "big") + encoded_msg
+        timestamp = int(time.time_ns() / 1000) # UNIX timestamp in microseconds
+        log_msg = self.frame_type + len(encoded_msg).to_bytes(4, "big") + timestamp.to_bytes(8, "big") + encoded_msg
```

Comment:
This is a breaking change, does it mean that older images with older rie do not work with this version of ric?
Shouldn't we keep previous packet structure as a fallback option if interface counterpart cannot support the latest version?

Reply:
Disregard that, my mistake. It is not related to https://github.com/aws/aws-lambda-runtime-interface-emulator

Reply:
It's not breaking change - the consumer of the messages can process both new and old formats.

Reply:
Yep, cool, LGTM